EIPS/eip-4938.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 4938
+title: "eth/67: Removal of GetNodeData"
+author: Marius van der Wijden <marius@ethereum.org>, Felix Lange <fjl@ethereum.org>, Gary Rong <garyrong@ethereum.org>
```

Comment:
```suggestion
author: Marius van der Wijden @MariusVanDerWijden , Felix Lange <fjl@ethereum.org>, Gary Rong <garyrong@ethereum.org>
```
We require at least one of the authors to use a github handle so that the bot can tag them.